source/index.txt

```diff
@@ -1,10 +1,102 @@
-==================
-Kotlin Sync Driver
-==================
+==========================
+{+driver-long+}
+==========================
 
-.. default-domain:: mongodb
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
 
-Your words here. Don't forget to add a toctree!
+   /quick-start
+   /quick-reference
+   /whats-new
+   /usage-examples
+   /faq
+   /connection-troubleshooting
+   /issues-and-help
+   /compatibility
+   View the Source <https://github.com/mongodb/mongo-java-driver>
 
-Have a lovely day!
+Introduction
+------------
 
+Welcome to the documentation site for the {+driver-long+}, the official
+MongoDB driver for synchronous Kotlin applications. Download the driver by using
+`Maven <https://maven.apache.org/>`__ or `Gradle <https://gradle.org/>`__, or set up a
+runnable project by following our Quick Start guide.
+
+.. tip:: Other Kotlin Platforms for MongoDB
+
+   If your Kotlin application requires asynchronous processing, use the
+   :driver:`Coroutine Driver </kotlin/coroutine/currenet>`, which uses coroutines for
+   server-side applications.
+   
+   If you are developing an Android or Kotlin Multiplatform (KMP)
+   application, you can use the :realm:`MongoDB Atlas Device Kotlin SDK </sdk/kotlin/>` 
+   to access Atlas App Services and to manage your Realm data.
```

Comment:
C: I believe Atlas App Services and the Atlas Device SDKs are two separate products, based on what Kyle has said to me, but I'll also defer to Rea on that one.

Reply:
good point, I think we can reword as 
```suggestion
   to implement Device Sync and to manage your Realm data.
```